src/components/RangeSlider/RangeSlider.tsx

```diff
@@ -286,6 +286,8 @@ export const RangeSlider = ({
 
   const domain = max - min;
 
+  const handleEventWithAnalytics = useAnalytics();
+
   // set the drag value asynchronously at a lower frequency for better performance
   const valueBuffer = useRef(0);
   const debouncedDrag = debounce(() => onDrag(valueBuffer.current), debounceInterval);
```

Comment:
This would be good to add analytics too as well - if you want to just get the base events in and make a PR, feel free to create a new issue with any events you didn't get to for someone else to follow up with.

Reply:
I added analytics to these events I had skipped over: ![image](https://user-images.githubusercontent.com/39521214/130681412-d53778f5-46a4-4a79-a371-638ea47d1e9a.png)
Let me know if you see any others I missed